components/developer/gcc-11/Makefile

```diff
@@ -18,12 +18,13 @@
 BUILD_BITS=64
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_VERSION= 11.2.0
-COMPONENT_REVISION= 1
+COMPONENT_VERSION= 11.3.0
 COMPONENT_SRC= gcc-releases-gcc-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE= $(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.gz
 COMPONENT_ARCHIVE_URL=	https://github.com/gcc-mirror/gcc/archive/releases/$(COMPONENT_ARCHIVE)
-COMPONENT_ARCHIVE_HASH=	sha256:5efe343a97e95fe9031aa23e103755ec3bc56855c865c4b6ffcbad6c25675514
+COMPONENT_ARCHIVE_HASH=	sha256:5b117751dd46ebfe9774da9efc977c094f1e0a7061c887d5f8b3fe00725186d2
+
+COMPONENT_BUILD_ARGS=	-j20
 
```

Comment:
Please remove this. We set the build parallelism in the jenkins job.

Reply:
Okay, will remove COMPONENT_BUILD_ARGS = j20